packages/frontend-2/components/header/nav/UserMenu.vue

```diff
@@ -65,6 +65,19 @@
               </NuxtLink>
             </MenuItem>
           </div>
+          <div class="border-t border-outline-3 py-1 mt-1">
+            <MenuItem v-slot="{ active }">
```

Comment:
Not blocking, but maybe we could make this a bit more secondary in the menu, not sure how, but maybe @michalspeckle has an idea

Reply:
I like that 😄